.github/workflows/issue_commands.yml

```diff
@@ -17,8 +17,14 @@ jobs:
           ref: main
       - name: Install Actions
         run: npm install --production --prefix ./actions
+      - name: "Generate token"
+        id: generate_token
+        uses: tibdex/github-app-token@b62528385c34dbc9f38e5f4225ac829252d1ea92
```

Comment:
what's this?

Reply:
[This action](https://github.com/tibdex/github-app-token) turns the private key from the app into a token that's usable for running the issue_commands.  But I actually don't know why the recommendation was to use that particular commit.  This is a good callout!  I will ask